### ISSUE TITLE
"Open File" does not have keyboard focus on Gnome shell

### ISSUE THREAD

**Issue by hongquan** [Microsoft/vscode#40631]
- VSCode Version: 1.19.1
- OS Version: Ubuntu 17.10 (with GNOME Shell desktop environment)

### Steps to Reproduce:

1. From VS Code, press Ctrl + O to open FileChooser dialog

### What is expected:
I can use arrow keys to select files.

### What actually happened:
The dialog lost focus, so I couldn't use arrow keys, before using mouse to "select" the dialog.

Reproduces without extensions: No
Note: Even I uninstalled all extension, this still happened. Not happen if launch with `code --disable-extensions`

**Comment by bpasero** [comment]
@hongquan once you click into the dialog, keybindings work?

**Comment by hongquan** [comment]
@bpasero Yes, once I click into the dialog, keybindings work.

**Comment by ericman314** [comment]
Same issue here. I have ilich8086.coldfusion@0.0.11 installed. Running with `--disable-extensions` causes the error not to be reproduced.

Ubuntu 17.10
Visual Studio Code Version 1.19.0

This is quite bad because I press Ctrl+O to open the file dialog, and by habit begin to type the first characters of the file I want to open. Instead, those characters get inserted into the file I was editing without me realizing. I'm still finding little snippets of filenames all over.

Apart from that, I'm very impressed with the editor. Well done!

**Comment by jenb** [comment]
Appears fixed for me in Insiders 1.20.0 on Debian  (I could reliably repro it on 1.19.0).

**Comment by ericman314** [comment]
I'm on 1.20.1 now and it appears to be fixed for me too.

**Comment by bpasero** [comment]
@hongquan can you confirm that this works again? Note though that I cannot remember a change on our end that could have an impact so it may have been fixed via a Linux update?

**Comment by hongquan** [comment]
I works again.
